include/openmc/simulation.h

```diff
@@ -47,6 +47,8 @@ extern const RegularMesh* ufs_mesh;
 extern vector<double> k_generation;
 extern vector<int64_t> work_index;
 
+extern std::vector<double> bins_pht;
```

Comment:
You shouldn't make these extern declarations twice.  It appears you should only have them in simulation.h. If you need them in settings.cpp, you should include simulation.h there.

Reply:
I think that's part of the problem: my implementation doesn't yet work properly with OpenMC's tally functions. I think I need to work on how the tally output is determined in the end from the pht_storage array. Then we don't need to define 
external std::vector<double> bins_pht; 
nor
external int cell_pht;
here.
I think I was in too much of a hurry here overall and really wanted to create a reasonably running version.